src/retasc/__main__.py

```diff
@@ -22,6 +22,11 @@ def parse_args():
         "-v", "--version", action="version", version=f"%(prog)s {__version__}"
     )
 
+    parser.add_argument("--jira-url", action="store", help="Jira API URL")
+    parser.add_argument(
+        "--jira-token", action="store", help="Jira authentication token"
+    )
+
```

Comment:
I would be better to drop this for now since it is unused and because we will likely use environment variables to configure the service.

```suggestion
```